plugin/trino-pinot/pom.xml

```diff
@@ -382,6 +384,13 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.pinot</groupId>
+            <artifactId>pinot-yammer</artifactId>
+            <version>${dep.pinot.version}</version>
+            <scope>runtime</scope>
```

Comment:
shall we make a noop implementation for this pinot-metrics interface ?

Reply:
> how are we getting this during runtime?

Required by the PinotQueryClient which requires PinotMetricsFactory. Would it be ok to leave it, so that we can have broker metrics? The worker functions like a broker without a reducer service (soon we will put that in also). wdyt?